store/storetest/user_store.go

```diff
@@ -2950,6 +2948,18 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 		etag4 := result.Data.(string)
 		require.Equal(t, etag3, etag4, "etag should not have changed")
 	})
+
+	// Add u3 to team 2
+	store.Must(ss.Team().SaveMember(&model.TeamMember{TeamId: teamId2, UserId: u3.Id}, -1))
+	u3.UpdateAt = store.Must(ss.User().UpdateUpdateAt(u3.Id)).(int64)
+
+	t.Run("etag for profiles not in team 1 after u3 added to team 2", func(t *testing.T) {
+		t.Skip()
```

Comment:
Can you add a comment here explaining why we're skipping, and that the etag is still changing right now even though it wouldn't have to with a perfect hashing solution?